one-ui/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -1,41 +1,87 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, cleanup, fireEvent } from '@testing-library/react';
 import SourceToEntityMap from './source-to-entity-map';
 import data from '../../../../config/data.config';
+import { shallow } from 'enzyme';
+import SplitPane from 'react-split-pane';
 
 describe('Source to Entity mapping component', () => {
-  let wrapper: any;
+    afterEach(cleanup);
+    test('Modal renders properly', () => {
+        const { getByTestId,  getByText, getByRole } = render(<SourceToEntityMap {... {mapData: data.mapProps.mapData, entityTypeTitle : data.mapProps.entityTypeTitle, sourceData: []}} mappingVisible={true}/>);
+        expect(getByText('Source Data')).toBeInTheDocument();
+        expect(getByText('Test')).toBeDisabled;
+        expect(getByText('Clear')).toBeDisabled;
+        expect(getByTestId("entityContainer")).toBeInTheDocument();
+        expect(getByTestId("srcContainer")).toBeInTheDocument();
+        expect(getByText("Unable to find source documents using the specified collection or query.")).toBeInTheDocument;
+        expect(getByTestId("srcContainer")).toHaveClass("sourceContainer");
+        expect(getByText('Entity: Person')).toBeInTheDocument();
+        expect(getByRole('presentation').className).toEqual("Resizer vertical ");
+    });
 
-  beforeEach(() => {
-    wrapper = shallow(<SourceToEntityMap {...data.mapProps} />);
-  });
+    test('Modal renders properly',  () => {
```

Comment:
Can we update the test description to be unique so we can debug in case of test failure

Reply:
Changed the description